.github/workflows/check-pr.yml

```
@@ -62,7 +42,7 @@ jobs:
           script: yarn pretify
```

```suggestion
          script: yarn prettify
```

Хочется везде поправить 😅